buildenv/jenkins/common/pipeline-functions.groovy

```diff
@@ -806,7 +806,7 @@ def move_spec_suffix_to_id(spec, id) {
     def spec_id = [:]
     spec_id['spec'] = spec
     spec_id['id'] = id
-    for (suffix in ['aot', 'cm', 'jit', 'valhalla', 'uma', 'ojdk292', 'vt_standard']) {
+    for (suffix in ['aot', 'cm', 'jit', 'valhalla', 'uma', 'ojdk292', 'vt_standard', 'criu']) {
```

Comment:
Pls put the whole list in alphabetical order

Reply:
updated